routes/api/contacts.js

```diff
@@ -120,6 +125,38 @@ router.put("/:contactId", async (req, res, next) => {
     }
   } catch (err) {
     console.log(err.message);
+    res.send(err);
+  }
+});
+
+router.patch("/:contactId/favorite", async (req, res, next) => {
+  try {
+    const body = {
+      favorite: req.query.favorite,
+    };
+    const result = await updateStatusContact(req.params.contactId, body);
```

Comment:
Ja bym walidację przeniósł z tamtej funkcji do linii 136 i będzie to prostsze w zrozumieniu co się dzieje.